cve_bin_tool/cli.py

```diff
@@ -12,35 +12,46 @@
 
 # Python 2 compatibility stuff
 from __future__ import print_function
-import sys
+
+import argparse
+import importlib.util
+import logging
+import multiprocessing
 import os
-import csv
 import platform
 import subprocess
-import logging
-import argparse
+import sys
 import textwrap
 import threading
-import pkg_resources
-import multiprocessing
 from collections import defaultdict
 
-from .version import VERSION
-from .util import DirWalk, inpath
-from .extractor import Extractor
-from .strings import Strings
-from .file import is_binary
-from .OutputEngine import OutputEngine
+import pkg_resources
 
+from .OutputEngine import OutputEngine
 from .cvedb import CVEDB, OLD_CACHE_DIR
+from .extractor import Extractor
+from .file import is_binary
 from .log import LOGGER
+from .strings import Strings
+from .util import DirWalk, inpath
+from .version import VERSION
 
 try:
     import queue
 except ImportError:
     import Queue as queue
 
 
+egg_updater_path = os.path.join(os.path.dirname(__file__), "egg_updater.py")
+if os.path.isfile(egg_updater_path):
+    spec = importlib.util.spec_from_file_location("egg_updater", egg_updater_path)
+    egg_updater = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(egg_updater)
+else:
+    egg_updater = None
+DEVELOP = True
```

Comment:
This block isn't necessary now that we've moved the module into the library, we can import with `from . import egg_updater`

Reply:
I have did it.